source/tyr/tyr/resources.py

```diff
@@ -979,6 +979,14 @@ def put(self, version=0, id=None, name=None):
             default=instance.pt_planners_configurations,
         )
 
+        parser.add_argument(
```

Comment:
it seems you have parsed the argument but it's not assign to attributed in the instance: https://github.com/hove-io/navitia/pull/4361/files#diff-54a917ad57e267be1145c2558782c533a4e302c09b98fc576982a12ea32045c7R1260